ark/phenotyping/som_utils.py

```diff
@@ -391,13 +445,19 @@ def create_fov_pixel_data(fov, channels, img_data, seg_labels,
     seg_labels_flat = seg_labels.flatten()
```

Comment:
When making column_index above, it should actually be np.tile(range(img_data.shape[1], img_data.shape[0]). Probably not causing issues so far because we've been working with square images.